index.html

```diff
@@ -5,20 +5,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome Books</title>
-    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="index.css" />
   </head>
   <body>
     <div class="collection" id="book-collection">
       <h2 class="book-heading">Awesome Books</h2>
       <div id="book-list"></div>
```

Comment:
- [ ] Please make sure the main heading is aligned in the center to follow the wireframe design. Currently, it's aligned to the left.

### Your design

![image](https://user-images.githubusercontent.com/83803414/218804350-96405824-6891-4c4e-901b-61e9d2ea478b.png)

### Wireframe design

![image](https://user-images.githubusercontent.com/83803414/218804508-fa59a97a-17a5-43fb-9ea0-91029afc7d9b.png)